### ISSUE TITLE
Pokeball Priority

### ISSUE THREAD

**Issue by AyameNoop** [finndev/PokeBuddy#743]
Is it just me or does the bot seem to use pokeballs in order from best (Ultraball) to worse (normal Pokeball)

Is there any way to change the priority to use regular pokeballs first?

**Comment by Thorben24** [comment]
No, this is fixed already. It takes usually PokeBalls, if the CP is higher, the catch probability lower, it will auto use greater ball or ultra ball.

**Comment by AyameNoop** [comment]
yea i was searching and just saw it
